examples/src/test/scala/docs/kafka/KafkaDocExample.scala

```diff
@@ -163,8 +166,28 @@ object KafkaDocExample {
         projectionId,
         sourceProvider,
         () => sessionProvider.newInstance(),
-        handler = () => new WordCountHandler(projectionId))
+        handler = () => new WordCountSlickHandler(wordRepository))
     //#exactlyOnce
+
+    // #exactly-once-jdbc-handler
+    class WordCountSlickHandler(val wordRepository: WordRepository)
```

Comment:
Shouldn't this be the `WordCountJdbcHandler`?

Reply:
correct, wrong naming

Reply:
For a second, I thought that I was getting crazy because I have read WordCountJdbcHandler in that PR. 

It turns out there are two.